src/jio-report-a-problem.ts

```diff
@@ -65,34 +71,27 @@ export class ReportAProblem extends LitElement {
     const url = this.url || this.locationHref;
     const title = this.pageTitle || this.windowTitle;
 
-    const queryParams = new URLSearchParams();
-    queryParams.append('labels', 'bug');
-    queryParams.append('template', '4-bug.md');
-    queryParams.append('title', `${title} page - TODO: Put a summary here`);
-    queryParams.append('body', outdent`
+    if (this.usePluginSiteReportUrl) {
+
+      const queryParams = new URLSearchParams();
+      queryParams.append('labels', 'bug');
+      queryParams.append('template', '4-bug.yml');
+      queryParams.append('title', `${title} - TODO: Describe what's wrong`);
```

Comment:
I would omit this so the title field will be empty and people would have to actually fill it

Reply:
Probably in another PR though